### PR TITLE
Feature/browser

### LINKED ISSUE #14: web browser tool
AIvil can web surfing.

Person can do the following actions.
- scroll down
- click
- read
- write in textarea or content editable div tag

### PR DESCRIPTION
**Pull request by adldotori**
Browser Tool can do this:
open: url
scroll: position
move: css selector
click: css selector
write: {css selector: input}
close: empty>

Ex.
image

It's not perfect, but it's working, and I'll improve this tomorrow.

Test prompt: Go to https://app.adcio.ai/en/ and log in to <your_id> / <your_password>
Test prompt: Please search what's the most delicious apple on Google
image

- resolved #14

### CODE REVIEW

**Comment by JoonHong-Kim** [comment]
Show me later~

File: core/civilization/person/tool/browser.ipynb
Comment (JoonHong-Kim) [inline core/civilization/person/tool/browser.ipynb:1]: 이거 냅두는거 맞나요? ㅋㅋㅋ

File: aivilization/main.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 
 
 def main():
-    civilization = Civilization(default_tracers=[LogTracer, RedisTracer])
+    civilization = Civilization(default_tracers=[LogTracer])
```
Comment (hanchchch) [inline aivilization/main.py:10]: how about this

```suggestion
    default_tracers = [LogTracer]
    if settings.REDIS_HOST:
        default_tracers.append(RedisTracer)
    civilization = Civilization(default_tracers=default_tracers)
```

File: core/civilization/person/tool/browser.ipynb
Comment (adldotori) [reply core/civilization/person/tool/browser.ipynb:1]: umm it's for your testing. if it doesn't exist, debugging is so hard

File: core/civilization/person/action/base.py [hunk 2]
```diff
@@ -39,16 +39,16 @@ class ActionType(Enum):
...
-            description=(
-                "Build or rebuild a reusable tool when you can't do it yourself. "
-                "It must have stdout, stderr messages. "
-                "It should be executable with the following schema of commands: `python tools/example.py instruction extra`"
-            ),
-            name="Tool's Name (snake_case)",
-            instruction="Tool's description that includes objective, instruction format, extra format, output format",
-            extra='Python Code for Building Tools (format: ```pythonprint("hello world")```)',
-        ),
+        # ActionProperty(
```
Comment (hanchchch) [inline core/civilization/person/action/base.py:42]: ?????

File: core/civilization/person/tool/browser.ipynb
Comment (JoonHong-Kim) [reply core/civilization/person/tool/browser.ipynb:1]: got it

**Review by hanchchch — COMMENTED** [review]
wow

File: core/civilization/person/action/base.py (hunk 2 above)
Comment (adldotori) [reply core/civilization/person/action/base.py:42]: build is too hard I want to remove it before AIvil can build stable.

File: aivilization/main.py (hunk 1 above)
Comment (adldotori) [reply aivilization/main.py:10]: good~

**Pull request merged**